ci/mkpipeline.py

```diff
@@ -218,11 +298,11 @@ def set_default_agents_queue(pipeline: Any) -> None:
             and "group" not in step
             and "trigger" not in step
         ):
-            step["agents"] = {"queue": "linux-aarch64-small"}
+            step["agents"] = {"queue": DEFAULT_AGENT}
 
 
 def check_depends_on(pipeline: Any, pipeline_name: str) -> None:
-    if pipeline_name != "test":
+    if pipeline_name not in ("test", "nightly", "release-qualification"):
```

Comment:
Is `nightly` here correct and if so, why does it differ from `nightlies` in Buildkite?

Reply:
Yes, it's a mess because the Buildkite pipeline is called nightlies, but the directory is ci/nightly. We should unify those by having a symbolic link for a while, then switching over to the new name.